### PR TITLE
Add/bunch of brand icons

### PR DESCRIPTION
**Pull request by j-mnizhek**
## Motivation and Context

Added 12 new brand icons:
`InstagramColored`, `ShopifyColored`, `WooCommerceColored`, `MetaColored`, `WhatConvertsColored`, `CallTrackingMetricsColored`, `GoogleDisplayVideoAds`, `PipedriveColored`, `CampaignMonitorColored`, `ConstantContactColored`,`Yext`, `BirdeyeColored`.

## How has this been tested?

Manually.

## Screenshots (if appropriate):
image
image
Screenshot 2024-10-17 at 12 47 12
Screenshot 2024-10-17 at 12 47 01
Screenshot 2024-10-17 at 12 46 51
Screenshot 2024-10-17 at 12 46 40
Screenshot 2024-10-17 at 12 46 30
Screenshot 2024-10-17 at 12 46 19
Screenshot 2024-10-17 at 12 46 06
Screenshot 2024-10-17 at 12 45 36
Screenshot 2024-10-17 at 12 45 12
Screenshot 2024-10-17 at 12 41 30

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [x] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/icon/svg/icon/Yext/l.svg
Comment (j-mnizhek) [inline semcore/icon/svg/icon/Yext/l.svg:1]: Wow, interesting why Yext icon doesn't show up there 🤔
image

Comment (sheila-semrush) [reply semcore/icon/svg/icon/Yext/l.svg:1]: ahaha the stroke must be too thin for the renderer

Comment (sheila-semrush) [reply semcore/icon/svg/icon/Yext/l.svg:1]: yup, it's less than 1px (in the L size just slightly less than 1px), so it must round it down to 0

File: website/docs/style/icon/icons-list.js [hunk 1]
```diff
@@ -4328,6 +4476,22 @@ const iconsList = {
...
     },
+    // {
+    //   name: 'InstagramColored',
+    //   size: ['l', 'm'],
+    //   group: 'Color',
+    //   tags: [
+    //     'instagram',
+    //     'social',
+    //     'logo',
+    //     'facebook',
```
Comment (sheila-semrush) [inline website/docs/style/icon/icons-list.js:4487]: if we added "facebook" here, maybe also add "meta"?

Comment (j-mnizhek) [reply website/docs/style/icon/icons-list.js:4487]: Good point! I'll add

File: semcore/icon/svg/icon/Yext/l.svg
Comment (j-mnizhek) [reply semcore/icon/svg/icon/Yext/l.svg:1]: Lol 😆 I'll try to do something with this guy

**Review by sheila-semrush — APPROVED** [review]
Left a couple of small suggestions in Figma and here, in general everything's good!

**Comment by j-mnizhek** [comment]
Finally! I've added InstagramColored icon and updated Yext icon.

**Review by Valeria-Zimnitskaya — APPROVED** [review]

**Pull request merged**